js-exercises/bind-method/bindMethod.js

```diff
@@ -0,0 +1,12 @@
+function bind(fn, obj) {
+  function newFn(value) {
+    const objValue = Object.values(obj)[0];
+    if (typeof value !== 'number') {
+      return null;
+    }
+    return objValue + value;
```

Comment:
We are not invoking `fn` at all.
We need to call `fn` while setting its thisArg/context to the `obj`.
This may pass the given test cases, but this is not satisfying the requirement at all